tests/unittests/algo/test_dehb.py

```diff
@@ -0,0 +1,126 @@
+"""Perform integration tests for `orion.algo.dehb`."""
+
+import itertools
+
+import numpy
+import pytest
+
+from orion.algo.dehb.dehb import IMPORT_ERROR, UnsupportedConfiguration
+from orion.core.utils import backward, format_trials
+from orion.testing.algo import BaseAlgoTests
+
+if IMPORT_ERROR:
```

Comment:
I don't really like this IMPORT_ERROR solution. Could you try using `pytest.importorskip` instead?
If it doesn't work, or makes the code ugly, then I guess leave this as it is.

Reply:
Not sure what is better. Otherwise we end up with something like this
```python
dehb_module = pytest.importorskip('orion.algo.dehb.dehb', reason='skipping DEHB tests')
UnsupportedConfiguration = dehb_module.UnsupportedConfiguration
```
I don't think that's an improvement over current version.

Reply:
I agree though, it's still uglier. Marking this as resolved.